README.md

```diff
@@ -43,6 +43,14 @@ That's it!
 
 We'd love to see contributions, feel free to fork! You should probably branch off ``master``, the branch ``release`` is used for stable revisions.
 
+Before committing, run the following commands:
+- `./manage.py test`
+- `./manage.py lint`
+- `./manage.py format`
+
+or, to combine all three, simply run `./manage.py precommit`
+
+To not have to do this manually all the time, we recommend you to set up `pylint` and `black` in your IDE.
```

Comment:
Maybe "We recommend you to set up ... in your IDE so you do not have to manually do this" or something?